egs/lre07/README.txt

```diff
@@ -1,9 +1,9 @@
  This directory (lre07) contains example recipes for the 2007 NIST
- Language Evaluation.  The subdirectory v1 demonstrates our single best LID
- system, which is an I-Vector based recipe.  Different LID systems combine
- easily to improve accuracy; contained in the subdirectory v2 is a fusion
- of the I-Vector-based system in v1 with an additional I-Vector-based system
- that using MFCC+Pitch features.
+ Language Evaluation.  The subdirectory v1 demonstrates the standard LID
+ system, which is an I-Vector based recipe using full covarience GMM-UBM and logistic regression model.  
+ The subdirectory v2 demonstrates the LID sysem using a Time Delay Deep Neural Network based UBM 
+ which is used to replace the GMM-UBM of v1. The DNN is trained using about 1800 hours of the english portion of Fisher.
```

Comment:
Please be mindful of typos here. E.g., covarience -> covariance, and sysem - > system. 

Also, please format the text so that lines don't go over 80 columns.